src/components/new-members/new-members.module.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .newUser {
-  pointer-events: none;
+  pointer-events: auto;
```

Comment:
Isn't the expectation, only super user should perform some action on pointer event and rest shouldn't?
Clarify in case I am thinking wrong here

Reply:
Yes any with super user role should be able to have cursor pointer property for rest of the user it should be default cursor pointer.